app/src/main/java/com/arctouch/codechallenge/util/Constants.kt

```diff
@@ -0,0 +1,15 @@
+package com.arctouch.codechallenge.util
+
+interface Constants {
```

Comment:
Here you don't really need to define `Constants` as interfaces, like we used to do in Java. Rather, you can declare `Constants` as object, a Singleton pattern implemented automatically by the Kotlin language:

    object Constants {
        const val API_KEY = "1f54bd990f1cdfb230adb312546d765d"
         const val ARGS_MOVIE = "movie"
         const val BACKDROP_URL = "https://image.tmdb.org/t/p/w780"
         const val BUNDLE_ARGS = "args"
         const val DEFAULT_LANGUAGE = "pt-BR"
         const val DEFAULT_REGION = "BR"
         const val PAGE_START: Long = 1
         const val POSTER_URL = "https://image.tmdb.org/t/p/w154"
         const val URL = "https://api.themoviedb.org/3/"
    }

Reply:
Thanks for to correct my constants file, I will follow this tips in a project that I'm working on. About the build variants, the best way is to separate in gradle because that way we can separate a production application with a test application, for example, you're right again.